docs/docs/deploying-to-kintohub.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Deploying to KintoHub
+---
+
+[KintoHub](https://www.kintohub.com) is a next-gen cloud package manager. Making it easy to host and manage websites, backend APIs, cron jobs, databases and more in a production grade fashion for all your apps.
+
+KintoHub has a free tier that you can use to deploy your static sites. Here are some features that come with it:
+
+- **CICD from GitHub** - You can build different branches and tag them to make sure the websites you deploy into staging and production environments are the same.
+- Automatic **[Let's Encrypt](https://letsencrypt.org) certificates.**
+- **Teams** - invite your collaborators on your workspace to work with you.
+- Setup **custom domain names** with automatic Let's Encrypt certificates.
+- Multiple **environments** (development, staging, production).
+- **Password Protection**.
+
+## Prerequisites
+
+This guide assumes you already have a Gatsby project to deploy. If you need a project, use the [Quick Start](/docs/quick-start) or use KintoHub's template [Gatsby Template](https://github.com/kintohub/gatsby-example) before continuing.
+
+## Setup
+
+In order to deploy a Gatsby site on KintoHub, you need to:
```

Comment:
This feels a bit misleading since the steps below walk you through the process of doing this. Perhaps removing the bullets and making it paragraph format will allow it to read more like a primer of what's to come?